package.json

```diff
@@ -23,6 +23,8 @@
     "build": "grunt default && gulp default",
     "lint": "eslint src/ && grunt lint && gulp lint",
     "testonly": "./resources/node_test.sh",
+    "testmocha": "mocha --require ts-node/register --recursive 'test/*.ts'",
```

Comment:
Sounds good. Would you settle for something like `*.test.ts`? Feels just a little bit cleaner to me. Otherwise, I'd get the feeling that 'test' is part of the filename instead of telling you what the file is, if that makes sense.